src/Accordion.php

```diff
@@ -159,20 +163,24 @@ public function renderItems()
      * @param string $header a label of the item group [[items]]
      * @param array $item a single item from [[items]]
      * @param int $index the item index as each item group content must have an id
+     * @param bool $expand whether or not to expand the item
      * @return string the rendering result
      * @throws InvalidConfigException
      * @throws \Exception
      */
-    public function renderItem($header, $item, $index)
+    public function renderItem($header, $item, $index, $expand)
```

Comment:
Can you also typify the parameters?

Reply:
Based on what [guideline](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md)?

Reply:
@xepozz that's Yii 2. We're not using types there.

Reply:
That's change of signature of public method. That could potentially result in:

1. Breaks when the method is called w/o passing additional parameter. That could be fixed by introducing default value such as `$expand = false`.
2. Breaks if there's class extending `Accordion`. That would result in "Fatal error: Declaration of Accordion::renderItem($header, $item, $index, $expand) must be compatible with Accordion::renderItem($header, $item, $index, $expand)".

If the introduction of this parameter cannot be avoided, the change can be released in 2.1.0 only.

Reply:
@simialbi `$this->expand` perhaps?

Reply:
I would prefer `$item['expand']`
```suggestion
    public function renderItem($header, $item, $index)
    {
         if (array_key_exists('content', $item)) {
             $id = $this->options['id'] . '-collapse' . $index;
             $options = ArrayHelper::getValue($item, 'contentOptions', []);
             $expand = ArrayHelper::getValue($item, 'expand', false); // <= Like this
```

Reply:
But with a deeper look it seems absurd. Get the item with the expand property to set it a bit further again. Maybe we should remove this check of expanded item:
```php
    $expanded = array_search(true, ArrayHelper::getColumn(ArrayHelper::toArray($this->items), 'expand', true));
    if ($expanded === false) {
        $expanded = current(array_keys($this->items));
    }
```
And replace it with some simpler check if there is a `expanded => true` item and if it's not, set the first one (for **BC**). That would even eliminate the need for setting the `expand` property in the loop (from my suggestion above).